### ISSUE TITLE
sourcemap points at incorrect directory

### ISSUE THREAD

**Issue by purge** [ivanhofer/typesafe-i18n#179]
### Version

4.1.3

### Describe the bug

contents of .map file point at packages/ directory which doesn't exist in shipped payload. causes crash in CRA4

from `node_modules/typesafe-i18n/runtime/esm/runtime/src/util.string.js.map`:
`{"version":3,"file":"util.string.js","sourceRoot":"","sources":["../../../../packages/runtime/src/util.string.ts"],`

### Reproduction

see above

**Comment by ivanhofer** [comment]
Oh I didn't know missing SourceMaps will cause something to crash 😅.
Fixed in version `4.1.4`

**Comment by purge** [comment]
Hey thanks for the quick update - while the maps aren't in the downloaded package they do appear to get generated again by the install process, though i can't work out exactly how this is happening (I had no idea esm / cjs packaging was so complicated these days). reproduction below

```
npm init --yes
npm i typesafe-i18n
find node_modules/typesafe-i18n/ -name "*.map" | wc -l
16
cat node_modules/typesafe-i18n/package.json | jq '.version'
"4.1.4"
```

**Comment by ivanhofer** [comment]
It really is complicated. A lot of hacks and workarounds are needed 🙈

I missed it in one place. Should be fixed in version `4.1.5`
